install_script.sh.template

```diff
@@ -1140,6 +1186,10 @@ if [ "$OS" == "RedHat" ]; then
       done
     fi
 
+    # Install the installer
+    # Note: this function will remove installed packages from the "packages" array
+    install_installer "$sudo_cmd" "$OS" "$DISTRIBUTION" "$DD_INSTALLER" "$DD_APM_INSTRUMENTATION_ENABLED" "packages" || true
```

Comment:
not specific to this callsite, but why do you use the variable name instead of the variable itself? (Please forgive my lack of shell scripting knowledge if this is a common pattern :sweat_smile: )

Reply:
Bash does not support passing reference or arrays as arguments. The usual way to deal with this is by passing the name of the variable containing the array and resolving the array in the function from its name. Agreed it's black magic for anyone not familiar with bash but it's the only way to avoid doing the filtering inline which I tried to avoid since we have 3 callsites

Reply:
TIL! Thanks for the explanation